### ISSUE TITLE
Enable GitHub Actions for code coverage.

### ISSUE THREAD

**Issue by BALaka-18** [BALaka-18/rake_new2#13]
## Description
Integrate the most appropriate GitHub Action for automatic generation of code coverage report on every code related PR made.

## Acceptance Criteria
- [ ] All instructions provided in the Description must be strictly followed.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Time Estimation
20 minutes

**Comment by jawsvk** [comment]
Hi @BALaka-18  I feel like taking this up, Would [Codecov](https://github.com/marketplace/actions/codecov) be the action that you are looking for?

**Comment by BALaka-18** [comment]
> Hi @BALaka-18 I feel like taking this up, Would [Codecov](https://github.com/marketplace/actions/codecov) be the action that you are looking for?

Yes @jawsvk , Codecov is by far the best. Assigning you the issue, make sure the PR is created after 1st Oct to be considered for Hacktoberfest.

**Comment by BALaka-18** [comment]
@jawsvk make sure your PR comes in on 1st October, because Code Coverage is vital before the code based PRs start pouring in.
